packages/herdtools7/herdtools7.7.56.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "herdtools7"
+version: "7.56.1"
+synopsis: "The herdtools suite for simulating and studying weak memory models"
+description: ""
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: [
+  "Luc Maranget <Luc.Maranget@inria.fr>"
+  "Jade Alglave <j.alglave@ucl.ac.uk>"
+]
+homepage: "http://diy.inria.fr/"
+bug-reports: "http://github.com/herd/herdtools7/issues/"
+doc: "http://diy.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/herd/herdtools7.git"
+build: ["sh" "./dune-build.sh" "%{prefix}%"]
+install: ["sh" "./dune-install.sh" "%{prefix}%"]
```

Comment:
Let’s see how it goes, but I expect this to fail. The sandbox does not allow to modify external folders. Also there is no library/executable specified by `public_name` for installation. I would recommend to use that and let dune generate the right `.install` file.